control/canfd_control/README.md

```diff
@@ -20,7 +20,7 @@ To build, you need to initialize and update `control/canfd_control/vendor/STM32C
 Alternatively, you can initialize and update only `control/canfd_control/vendor/STM32CubeH7` submodule after cloning, as shown below.
```

Comment:
```suggestion
Alternatively, you can initialize and update submodules after cloning, as shown below.
```

Reply:
Thanks!

Reply:
I thought about `ros2_packages` or `simulation`, but the previous description is confusing, so this seems better.